system/ee/ExpressionEngine/Controller/Utilities/ImportConverter.php

```diff
@@ -341,6 +342,8 @@ private function _datafile_to_array($file, $delimiter, $enclosure)
             }
         }
 
+       
+
```

Comment:
Can you remove these extra lines - just want to make sure small edits don't add any unnecessary changes.